### DISCUSSION TITLE
Welcome to Excalidraw Discussions!

### DISCUSSION THREAD

**Discussion by lipis** [excalidraw/excalidraw#2488] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mfkr** [comment]
Hi all, i'v tried run both excalidraw and excalidraw-room as container on my local envi without https, it's run well, but when i tried run them on k8s, i got these errors , is there any documentation relate to run the excalidraw with tls on?
![image (53)](https://github.com/excalidraw/excalidraw/assets/64786/dc7a394c-74c8-4e8d-9d17-b7a4ad0b0e64)

**Comment by lanamesquita** [comment]
Hello,
I discovered this project through [CodeTriage](https://www.codetriage.com/excalidraw/excalidraw) and would like to contribute by helping with labeling issues. Could you please guide me on how I can access this functionality? 😊

A bit about me:
I have been working in computing and systems development since 2012, holding various roles such as developer, tester, project manager, operations team member, requirements analyst, and UX designer. With a degree in Teleinformatics Engineering, I combine technical expertise with a focus on innovation, user experience, and accessibility. 🚀

I am eager to contribute to the Excalidraw project and help with its issue management process. Thank you for considering my request! 🙏

Best regards,
Lana Mesquita 💻✨
